Document/PostManager.php

```diff
@@ -25,6 +27,8 @@ class PostManager extends BaseDocumentManager implements PostManagerInterface
      * @param $slug
      *
      * @return mixed
+     *
+     * @deprecated NEXT_MAJOR: Will be removed, please use PostManager::findOneByPermalink instead
```

Comment:
No, the `NEXT_MAJOR` should be outside the `@deprecated`.

The `@deprecated` should have a proper message for end users.

```php
/**
 * @deprecated since version 3.x, to be removed in 4.0. Use PostManager::findOneByPermalink instead.
```